console-extensions.json

```diff
@@ -59,23 +59,14 @@
       "component": { "$codeRef": "AddressDetailsContainer.App" }
     }
   },
-  {
-    "type": "console.navigation/section",
-    "properties": {
-      "id": "activemq",
-      "perspective": "admin",
-      "name": "ActiveMQ Artemis",
-      "dataAttributes": { "data-quickstart-id": "qs-nav-home" }
-    }
-  },
   {
     "type": "console.navigation/href",
     "properties": {
       "id": "brokers",
-      "name": "ActiveMQ Artemis Self Provisioning",
+      "name": "Brokers",
       "href": "/k8s/all-namespaces/brokers",
       "perspective": "admin",
-      "section": "activemq"
+      "section": "workloads"
```

Comment:
This removes the top-level plugin section. Maybe we can keep it along with workloads so people can quickly find the spp plugin? (maybe adding an `About` item  to show versions, branding, doc links...)

Reply:
Yes that's a very good idea.

Reply:
We've just had a meeting with @Msarawan  looking a the options for the about page, in patternfly, about pages seems to be modals and not actual page. We need to discuss the topic in today's UX meeting. I think in the meantime we can merge @Msarawan's work, if we need to bring back the menu it'll be easy to do anyways.